src/app/trips/app-destinations-list/app-destinations-list.component.ts

```diff
@@ -1,23 +1,17 @@
 import { Component, Input } from "@angular/core";
-import { IDestination } from "../shared";
 
 @Component({
   selector: 'app-destinations-list',
   template: `
```

Comment:
The template should also be a template URL to point to an HTML file instead of inline strings.

Reply:
This is something Irwin mentioned to me at one point last week so I was able to apply this change before pushing PR#9